### DISCUSSION TITLE
limiting time frame for public group posts extraction? and extract by key word?

### DISCUSSION THREAD

**Discussion by tallevy22** [kevinzg/facebook-scraper#851] [Q&A]
1. is there a way to extract posts from a public group in a limited time frame? (lets say from year 2000-2020)
2. and is there a way to extract by keyword?

**Comment by neon-ninja** [comment]
No, the only way to get posts is to "scroll down" the page. Starting from most recent post, and going back. So you'll need to extract all posts, and filter by year/keyword yourself. I'd recommend using `options={"allow_extra_requests": False, "posts_per_page": 200}` to speed up the rate of post extraction
